tests-e2e/cypress/e2e/sdc-collection/stepThreeFilters.cy.ts

```diff
@@ -218,6 +218,55 @@ describe('SDC School Collection View', () => {
       });
     });
 
+    it('verifies special filters for ELL tab', () => {
+      cy.intercept(Cypress.env('interceptors').collection_students_pagination).as('pagination');
+      cy.visit('/');
+      cy.get(selectors.dashboard.dataCollectionsTile).click();
+      cy.get(selectors.dataCollectionsLanding.continue).contains('Continue').click();
+      cy.get('button[value="English Language Learning"]').click();
+
+      cy.get(selectors.ellComponent.tab).contains('Filters').click();
+      cy.get(selectors.activeFiltersDrawer.drawer).contains('1-5 years in ELL').click();
+      cy.get(selectors.activeFiltersDrawer.drawer).contains('Apply Filters').click();
+
+      cy.get(selectors.studentLevelData.detailsLoadingBar).should('exist');
+      cy.get(selectors.ellComponent.tab).find(selectors.studentLevelData.studentsFound).should('exist').contains(1);
+      cy.get(selectors.ellComponent.tab).find('tbody tr').each($cell => {
+        cy.wrap($cell).children().last().invoke('text').then((text) => {
+          expect(text).to.satisfy((value: string) => {
+            return value === '17-English Language Learning3';
```

Comment:
We expect the text to have a 3 at the end of it?

Reply:
Yes